keyboards/atreus/keymaps/dvorak_42_key/keymap.c

```diff
@@ -46,6 +81,8 @@ enum custom_keycodes {
 // building instructions:
 // make atreus:dvorak_42_key
 
+// flashing instructions:
+// avrdude -p atmega32u4 -c avr109 -U flash:w:atreus_astar_dvorak_42_key.hex  -P COM15
```

Comment:
You should be able to run `make atreus/astar:dvorak_42_key:flash` or `qmk flash -kb atreus/astar -km dvorak_42_key` instead of this, since the COM port is not guaranteed to be the same.

Reply:
ok, I changed the comment.